### PR TITLE
npm publish

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -123,6 +123,10 @@ _deploy_apex:
 	@if [ -e project.json ]; then $(call CONFIG_VARS,production) > $(APEX_PROD_ENV_FILE) && apex deploy --env-file $(APEX_PROD_ENV_FILE); fi
 	@if [ -e $(APEX_PROD_ENV_FILE) ]; then rm $(APEX_PROD_ENV_FILE) && $(DONE); fi
 
+npm-publish:
```
Comment (matthew-andrews) [inline Makefile]: `npm-publis%:`

in case you wanna override

Comment (matthew-andrews) [inline Makefile]: or possibly just `publis%`?

**Comment by matthew-andrews** [comment]
i dont feel strongly.  how do you feel `make publish` vs. `make npm-publish`?

**Comment by wheresrhys** [comment]
coudl add publish in future maybe, with npm-publish as an optional subtask of it.

It'd be nice to make it totally magic for components which have package.json#name starting with @financial-times... but still need a simple declarative way to publish those which don't follow that pattern yet

**Pull request merged**